readme.md

```diff
@@ -452,6 +452,8 @@ print_r($insertedPKs);
 // )
 ```
 
+**Note:** Due to the fact that `Faker` returns all the primary keys inserted, the memory consumption will go up drastically when you
```

Comment:
❓ 

Looks like we are missing something here!

😉

Reply:
Right, please fix this sentence